formbricks/formbricks
# fix: fixes empty in app surveys screen

## What does this PR do?



Fixes in app surveys empty responses UI



## Type of change




## How should this be tested?



- Make a new in-app survey without connecting an app.
- Go to the survey summary / responses page and check the UI

## Checklist
